Servers/WindowServer/WindowManager.h

```diff
@@ -141,7 +141,9 @@ class WindowManager : public Core::Object {
     const Gfx::Font& font() const;
     const Gfx::Font& window_title_font() const;
 
-    void set_resolution(int width, int height);
+    bool set_resolution(int width, int height);
+    int get_resolution_width();
+    int get_resolution_height();
```

Comment:
Please refrain from using `get_` prefixes in API's. Also it should be `const` as it does not mutate state.
I would also suggest using a `Gfx::Size` as the return type:

```cpp
Gfx::Size resolution() const;
```